modules/ROOT/pages/afm-create-monitor.adoc

```diff
@@ -66,6 +66,10 @@ After the monitor runs, click the *Monitored Endpoints* tab to see:
 
 As your monitor continues to run at the scheduled intervals, the statistics in this table are updated. They show the minimum, average, and maximum runtime for up to five executions of the test.
 
-Click *Tests* to see statistics and information pertaining to the monitor. Because you have only one test in the monitor, the statistics and information in this section apply to the test, too.
+Click *Tests* to see statistics and information pertaining to the monitor. Because you have only one test in the monitor that you just created, the statistics and information in this section apply to the test, too.
```

Comment:
Are "statistics" data about the monitor or about something else? 
If the former, consider "statistics and other information about the monitor." 
If the latter, can we specify what the statistics are about?

Reply:
Should the first reference to "the monitor" be to "your new monitor," given the addition of "that you just created"?